### DISCUSSION TITLE
How do you gainfully employ the Description field for simple transactions?

### DISCUSSION THREAD

**Discussion by d-tork** [firefly-iii/firefly-iii#8174] [General]
For years I've transcribed my receipts into these five columns, with an example transaction:
```
| Date     | Amount | Description | Category  | Location        |
| -------- | ------ | ----------- | --------  | --------------- |
| 1 Oct 23 | -25.00 | Whole Foods | groceries | Los Angeles, CA |
```
As I test out Firefly, I am mapping
* Description --> Expense Account
* Category --> Budget or Category
* Location --> Tags

That leaves me with an empty Description field in Firefly for _most daily transactions_. I mean what more can I say about this withdrawal to pay Whole Foods (destination account) for my groceries (budget/category) in a specific place (a tag, so when the location feature is finished I can convert them all to gps)? I have no more to say about this, yet the Description field is required. 

In other cases I can think of a use for it, like `destination account: Venmo | category: social | description: paid John for drinks at The Bar`. Indeed, most of the examples in the documentation use the Description field to describe rare and specific circumstances like "money back for fancy lunch", "cash for going out", or "lost a note of 5". Or at least _I'm_ not having fancy lunches all the time and losing money on the regular. 

So for your ordinary trip to the grocery store when you don't get robbed or split the bill with a friend--and you are the type of person who is using expense accounts for named businesses, rather than broad categories--what are you putting in the Description field to satisfy that requirement?

**Comment by JC5** [comment]
I use the shop name for the destination account and a generic description for the description, ie "Groceries". The demo site shows it fairly well.

**Reply by d-tork** [reply]
Isn't that redundant, since groceries is one of the best examples of a budget? Making a category isn't a bad idea either 

I suppose I could make rules to expedite future data entry, i.e. `Description: groceries | Destination: Whole Foods | Amount: XX` and let the rules select the groceries budget and category for the transaction. But at the end of the day, the database still has "groceries" in two or three fields which feels wrong as a programmer and the DBA of my own stuff.

**Reply by JC5** [reply]
It's entirely up to you. For me the category is "daily groceries" and yes the budget is groceries but many other things are also groceries.
